tests/integration/ceph_base_mgr_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	cephclient "github.com/rook/rook/pkg/daemon/ceph/client"
+	"github.com/rook/rook/tests/framework/utils"
+)
+
+type MgrCLI struct {
+	k8sh      *utils.K8sHelper
+	namespace string
+}
+
+func (cli MgrCLI) Execute(command []string) (error, string) {
```

Comment:
To run a `ceph` command we will want to do it in the toolbox that is running as part of the integration test setup. You can follow the pattern in the [waitForFilesystemActive()](https://github.com/rook/rook/blob/b33804d0345ffd344b00638fc4259728f774339c/tests/integration/ceph_base_file_test.go#L341) test method that calls some test helper methods that will actually execute the commands in the toolbox. So if you think it is useful you can refactor that method to execute any ceph commands from the tests. The helper should probably go in tests/framework/installer/ceph_installer.go

Reply:
OK.